docs/boot-options.rst

```diff
@@ -682,19 +682,6 @@ back to a local boot.
 Storage options
 ---------------
 
-.. inst.nodmraid:
-
-inst.nodmraid
```

Comment:
I guess this should be moved to the [Removed Options](https://anaconda-installer.readthedocs.io/en/latest/boot-options.html#removed-options).

Reply:
Ok, I hope the comment about the removal makes sense.